src/main/java/duke/Response.java

```diff
@@ -0,0 +1,43 @@
+package duke;
+
+/**
+ * A duke.Response from the application to some user input.
```

Comment:
I think there's no need to include the package of the `Response` class.
```suggestion
 * A Response from the application to some user input.
```
Same goes for the other JavaDoc comments in this file.